invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/files.html

```diff
@@ -107,16 +121,20 @@ <h3 class="active title panel-heading {{ record.ui.access_status.id }} m-0">
             <div>
               <a href="{{ file_url_download }}">{{ file.key }}</a>
             </div>
+            {% if file.checksum is defined %}
             <small class="ui text-muted font-tiny">{{ file.checksum }}
             <div class="ui icon inline-block" data-tooltip="{{_('This is the file fingerprint (checksum), which can be used to verify the file integrity.')}}">
               <i class="question circle checksum icon"></i>
             </div>
             </small>
+            {% endif %}
           </td>
+          {% if file.size is defined %}
           <td>{{ file.size|filesizeformat(binary=binary_sizes) }}</td>
+          {% endif %}
           <td class="right aligned">
             <span>
-              {% if with_preview and file_type|lower is previewable %}
+              {% if with_preview and file_type|lower is previewable and not is_remote_file %}
```

Comment:
is_remote_file needs to be declared before use in this macro, it is undefined

Reply:
Good catch! I fixed it